cn.navclub.ldbfx.json

```diff
@@ -0,0 +1,42 @@
+{
+    "app-id": "cn.navclub.ldbfx",
+    "runtime": "org.gnome.Sdk",
+    "runtime-version": "42",
+    "sdk": "org.gnome.Sdk",
+    "command": "ldbfx",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--filesystem=home"
```

Comment:
Not a blocker, but consider using GtkFileChooserNative so your app uses file portal for accessing sqlite files.